inference/models/__init__.py

```diff
@@ -0,0 +1,24 @@
+try:
```

Comment:
What is happening here?

Reply:
`clip` `sam` and `gaze` are all optional models so a user might not have the dependancies needed to import them.

Reply:
Sounds good!